### DISCUSSION TITLE
An unexpected error occurred while connecting Node

### DISCUSSION THREAD

**Discussion by Aenoxic** [EvieePy/Wavelink#267] [General]
An unexpected error occurred while connecting Node(identifier=<HIDDEN>, uri=<HIDDEN>, status=NodeStatus.CONNECTING, players=0) to Lavalink: "'Play' object has no attribute 'user'"
If this error persists or wavelink is unable to reconnect, please see: https://github.com/PythonistaGuild/Wavelink/issues

**Comment by chillymosh** [comment]
This is not a library issue, so I am converting to a Discussion.
Play is not a native class of Wavelink so you either have a typo or you have mishandled something in your code.
